common/src/main/java/io/homeassistant/companion/android/database/wear/FavoriteCachesDao.kt

```diff
@@ -0,0 +1,22 @@
+package io.homeassistant.companion.android.database.wear
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+@Dao
+interface FavoriteCachesDao {
+
+    @Query("SELECT * FROM favorite_cache where id = :id LIMIT 1")
+    fun get(id: String): FavoriteCaches
```

Comment:
There might not be a favorite for the provided ID

```suggestion
    fun get(id: String): FavoriteCaches?
```